hudi-utilities/src/main/java/org/apache/hudi/utilities/HoodieClusteringJob.java

```diff
@@ -109,6 +111,9 @@ public static void main(String[] args) {
     if (result == -1) {
       LOG.error(resultMsg + " failed");
     } else {
+      if (cfg.runSchedule) {
+        System.out.println("The schedule instant time is " + cfg.clusteringInstantTime);
```

Comment:
I think system.out.println will be better. Because println the instant time use system.out.println will output the string to spark stdout of UI.  If use LOG  by default the string will output to stderr with many other  logs.

Reply:
done